kubemarine/procedures/install.py

```diff
@@ -52,14 +52,29 @@ def system_prepare_check_system(cluster):
             raise Exception('%s host operating system is unsupported' % address)
         if context["os"]["family"] == "unknown":
             supported_os_versions = []
-            os_family_list = cluster.globals["compatibility_map"]["distributives"][context["os"]["name"]]
+            os_family_list = cluster.globals["compatibility_map"]["distributives"]
             for os_family_item in os_family_list:
-                supported_os_versions.extend(os_family_item["versions"])
-            raise Exception("%s running on unknown %s version. "
-                            "Expected %s, got '%s'" % (address,
-                                                       context["os"]["name"],
-                                                       supported_os_versions,
-                                                       context["os"]["version"]))
+                supported_os_versions.append(os_family_item)
+            if cluster.context['execution_arguments']['os_family'] in supported_os_versions:
+                cluster.log.debug("%s running on unknown %s version." % (address, context["os"]["name"]))
+                if context["os"]["name"] == 'ubuntu':
+                    context["os"]["family"] = 'ubuntu'
+                elif context["os"]["name"] == 'centos':
+                    context["os"]["family"] = 'centos'
+                elif context["os"]["name"] == 'rhel':
+                    context["os"]["family"] = 'rhel'
```

Comment:
if os.name is rhel, then os family can be either rhel or rhel8 https://github.com/Netcracker/KubeMarine/blob/main/kubemarine/resources/configurations/globals.yaml#L547